### PR TITLE
Document how to install on Fedora

### PR DESCRIPTION
**Pull request by m1ga**
Add `dnf` packages for Fedora.
~I need to double check with a vanilla Fedora installation because I already had some dev tools installed.~
Checked with a vanilla install and updated the script.

### CODE REVIEW

**Comment by m1ga** [comment]
ready to go!

File: README.md [hunk 1]
```diff
@@ -49,6 +49,13 @@ sudo apt install make gcc openjdk-8-jdk pkg-config meson zip \
                  libsdl2-dev
 ```
 
+For Fedora you need to install the following packages:
+```bash
+# enable RPM fusion free
+sudo dnf install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://download1.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm
```
Comment (rom1v) [inline README.md]: I have no experience with Fedora, but I have some questions.

> [RPM Fusion](https://rpmfusion.org/) provides software that the Fedora Project or Red Hat doesn't want to ship.

Which packages require RPM fusion?

Among them, which ones require the non-free repo?

File: README.md [hunk 2]
```diff
@@ -49,6 +49,13 @@ sudo apt install make gcc openjdk-8-jdk pkg-config meson zip \
                  libsdl2-dev
 ```
 
+For Fedora you need to install the following packages:
```
Comment (rom1v) [inline README.md]: (nitpicking) Please add an empty line (for consistency) ;)

File: README.md (hunk 1 above)
Comment (m1ga) [reply README.md]: ffms2-devel which contains libavformat

**Comment by m1ga** [comment]
I'll have a look at "rebase-ing". I though you could do this in the merge menu of github

File: README.md (hunk 2 above)
Comment (m1ga) [reply README.md]: added the new line

**Comment by rom1v** [comment]
> I'll have a look at "rebase-ing". I though you could do this in the merge menu of github

I'll do it for you if you want (you'll still be the author of the commit, I'll be the committer). ;)

**Comment by m1ga** [comment]
That would make it easier for me :+1:

File: README.md (hunk 1 above)
Comment (rom1v) [reply README.md]: OK. It should be in the free repo, so the nonfree is not necessary, is it?

Comment (m1ga) [reply README.md]: ah sorry, yes. I removed the non-free part!
https://www.rpmfind.net/linux/rpm2html/search.php?query=ffms2-devel

**Comment by rom1v** [comment]
Ah, another question: IIRC, the packages listed are for building with the prebuilt server.

Do you know the name of the package for java (for building the whole project)?

**Comment by m1ga** [comment]
If it works fine with openjdk then it is just `dnf install java`. That will install java-1.8.0-openjdk

**Comment by rom1v** [comment]
https://github.com/Genymobile/scrcpy/commit/70599998ebd72f1765eed9120693fac4e48471b9

**Pull request closed without merging**